articles/sql-or-nosql-when-to-choose-what/index.md

```diff
@@ -0,0 +1,123 @@
+
+It is not like the early times when data was stored on papers and cabinet. Data is now all around us. Information is the backbone of almost every modern business models. With the current steep growth of the database technology, as a developer building an application to store some data online, and already know what a database is, the big question you should be asking yourself is which database model will fit your application model. Knowing which database structure to use is critical to building outstanding applications.
+SQL or NoSQL? That is what we will dive in briefly, to understand their structures, explore them deeply, and understand how they work. These two databases do one thing, to store data, but the approach to do this may differ.
+
+### What SQL
+
+SQL is the most popular database structure. Which I too know at one instance heard of it. SQL (commonly pronounced as "see quell") stands for the structured query language.  SQL is a language used for writing query statements to interact and communicate with relational databases. A relational database is a database that has a tabular schema to store data in terms of rows, and columns. Each column contains data and each row is called a record. The data in the tables can be related to each other hence the 'relational database'. A collection of these multiple tables in a database is what we are calling a schema. A relation is simply a decorative phrase for a table.
```

Comment:
**Each column contains data and each row is called a record.**

"Each column contains data that are called attributes and each row is called a record." explains the point better.